tests/block_tools.py

```diff
@@ -1394,8 +1421,10 @@ def get_challenges(
     return cc_challenge, rc_challenge
 
 
-def get_plot_dir() -> Path:
-    cache_path = DEFAULT_ROOT_PATH.parent.joinpath("test-plots")
+def get_plot_dir(plot_dir: Optional[str] = None) -> Path:
```

Comment:
Should be Plot_dir: str="test-plots"

Reply:
What if plot_dir is None, this wont work

Reply:
It can't be none, we need test plots right?

Reply:
I meant what if the variable was left as default, which is none

Reply:
The default should be test-plots not None

Reply:
Ok so change it upstream too!

Reply:
Wait actually we already pass in test plots into init, so we can just save it as an instance variable and remove this one right?

Reply:
> Wait actually we already pass in plot dir into init, so we can just save it as an instance variable and remove this one right?

oh yeah lol, thats even better

Reply:
Oh wait, its not part of the block_tools class